internal/oci/runtime_oci.go

```diff
@@ -427,6 +435,32 @@ func (r *runtimeOCI) ExecContainer(ctx context.Context, c *Container, cmd []stri
 	return cmdErr
 }
 
+// moveExecToCgroup moves Exec command pid to the pod group
+func moveExecToCgroup(containerPid, commandPid int) error {
```

Comment:
1: is there any way we can merge this funcitonality with the CreateContainer variant?

2: I think we want this to be in `internal/config/cgmgr` so we can toggle based on cgroup type and version

Reply:
I'll look into this. Thanks for the suggestion!

Reply:
Moved the function to cgmgr.go. I don't see a way to merge this with CreateContainer, though